src/main/java/hw8/sections/ElementsCheckboxes.java

```diff
@@ -0,0 +1,44 @@
+package hw8.sections;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+
+import java.util.List;
+
+public class ElementsCheckboxes extends Section {
```

Comment:
Take a look at JDI typefield elements like Dropdown.

Reply:
Dropdown doesn't match the structure of the checkboxes section (No "select" tag, no caret to expand list). 
I've tried Checklist, but honestly, I couldn't make it work, because I don't understand at all how it works. Documentation didn't help.